gio2/ext/gio2/rb-gio2.c

```diff
@@ -31,4 +75,8 @@ Init_gio2 (void)
 
     rb_gio2_init_application(RG_TARGET_NAMESPACE);
     rb_gio2_init_pollable_source(RG_TARGET_NAMESPACE);
+
+    rbgobj_register_mark_func(G_TYPE_ACTION, rb_gio2_action_mark);
+    rbgobj_register_mark_func(G_TYPE_APPLICATION, rb_gio2_application_mark);
+    rbgobj_register_mark_func(G_TYPE_SIMPLE_ACTION_GROUP, rb_gio2_simple_action_group_mark);
```

Comment:
We don't need them because we care this in https://github.com/ruby-gnome/ruby-gnome/blob/master/gio2/lib/gio2/action-map.rb .